src/App.jsx

```diff
@@ -3,26 +3,48 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { TodosFromServer } from './app';
 
 class App extends React.Component {
   state = {
-    todos: [],
     selectedUserId: 0,
+    todos: [],
   };
 
+  componentDidMount() {
+    TodosFromServer().then(a => this.setState(
```

Comment:
Не називай з великої букви, вже багато разів про це писали, що потрібно з великої букви називати

Що таке `a` ?
```suggestion
    TodosFromServer().then(a => this.setState(
```